vignettes/GGIR.Rmd

```diff
@@ -532,6 +524,39 @@ use `GGIR`. So, if you would like to add annotations in the file, e.g.
 in the fourth column, then you will need to store it somewhere outside
 the output folder and explicitly point to it with `configfile` argument.
 
+### Tailoring GGIR to study protocol
+
+Usual study protocols to collect multiday accelerometer raw data last one
+week, either asking participants to wear accelerometers allday long (i.e., 24/7)
+or only the time they are awake. GGIR includes a number of arguments to 
+facilitate the processing of files collected with different study protocols. 
+For example:
+
+-   `strategy` - informs GGIR how to consider the design of the
+    experiment.
+    -   If `strategy` is set to value 1, then check out arguments
+        `hrs.del.start` and `hrs.del.end`.
+    -   If `strategy` is set to value 2, then GGIR will use the data
+        available between the first and the last midnight.
+    -   If `strategy` is set to value 3, then check out arguments
+        `ndayswindow`.
+    -   If `strategy` is set to value 4, then GGIR will use the data
+        from the first midnight until the very end of the file.
+-   `maxdur` - maximum number of days you expect in a data file based on
+    the study protocol.
+-   `AskedToWear247` - whether participants were instructed to wear the devices
+    during awake and sleep time or not. If `AskedToWear247 == FALSE`, then check
+    out arguments `part2ImpWithZero` and `part2ExcludeNonwear`.
+-   `part2ImpWithZero` - If `AskedToWear247 == FALSE`, this will be automatically set to 
```

Comment:
Please clarify how this interacts with the `do.imp` argument. It may be good to clarify both here and in the argument documentation for `part2ImpWithZero` what happens when `do.imp = FALSE` and `part2ImpWithZero == TRUE`.

Reply:
Clarified, then the metrics will not be imputed with zero, do.imp is the main argument to control doing (or not) imputation, as in the default settings.